buildenv/jenkins/JenkinsfileBase

```diff
@@ -158,6 +187,7 @@ def setupParallelEnv() {
 						booleanParam(name: 'IS_PARALLEL', value: false),
 						booleanParam(name: 'AUTO_DETECT', value: AUTO_DETECT),
 						string(name: 'TIME_LIMIT', value: "${TIME_LIMIT}"),
+						string(name: 'RUNTIME_NAME', value: currentDirectory),
```

Comment:
Is this `RUNTIME_NAME ` needed?

Reply:
Yes, that's the feature that was needed to distinguish one iteration of the same test from another. That's what fixed the issue with the iterations building over top of each other on the same machine.